org.eclipse.scout.rt.platform.test/src/test/java/org/eclipse/scout/rt/platform/BeanCreationWithExceptionsTest.java

```diff
@@ -346,6 +292,18 @@ private static IFuture<TestBean> scheduleGetBean() {
     return future;
   }
 
+  private static void assertTestBeanException(boolean expectedFromConstructor, BeanCreationException firstException, BeanCreationException secondException) {
+    try {
+      // assertion must match either for firstException
+      assertTestBeanException(expectedFromConstructor, firstException);
+    }
+    catch (AssertionError e) {
+      // or in rare cases for secondException (parts of second future may be run before parts of first future)
+      LOG.info("Fallback, check other exception", e);
```

Comment:
This is the same exception we already logged in assertTestBeanException - necessary? Or should output first and second exception instead?